tests/network/vmi_networking.go

```diff
@@ -688,14 +683,14 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 				var serverVMI *v1.VirtualMachineInstance
 
 				clientVMI = masqueradeVMI([]v1.Port{}, networkCIDR)
-				_, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(clientVMI)
+				clientVMI, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(clientVMI)
```

Comment:
very odd: to me, something like this has returned a missing-namespace error.

Reply:
Not the `Create` will return such an error, the usage after it in case you have not fetched the updated `vmi`.